freeipa.spec.in

```diff
@@ -17,11 +17,15 @@
     %global with_ipatests_option --without-ipatests
 %endif
 
+%if 0%{?with_python3:1}
+# with_python3 already defined
+%else
```

Comment:
You can make these three lines into one if you change the original condition to `%if 0%{!?with_python3:1}`